pkg/placement/plugins/addon/addon.go

```diff
@@ -82,7 +82,7 @@ func (c *AddOn) Score(ctx context.Context, placement *clusterapiv1beta1.Placemen
 		// get AddOnPlacementScores CR with resourceName
 		addOnScores, err := c.handle.ScoreLister().AddOnPlacementScores(namespace).Get(c.resourceName)
 		if err != nil {
-			klog.Warningf("Getting AddOnPlacementScores failed: %s", err)
+			klog.FromContext(ctx).Info("Failed to get AddOnPlacementScores", "error", err)
```

Comment:
Feels like we should use `klog.FromContext(ctx).Warn` instead of `Info` here to keep it the same as before?

Reply:
hey @xuezhaojun, warning function has been removed. Please take a look. [MigratingGuideline](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md)

![image](https://github.com/open-cluster-management-io/ocm/assets/72253189/483fc02c-6f4a-490a-89e7-76ac39b0c2cb)

Reply:
Got it! Thanks for explaining.